build/localnet/README.md

```diff
@@ -112,8 +112,11 @@ make localnet_client_debug
 
 You can find private keys and addresses for all actors in the [private-keys.yaml](./manifests/private-keys.yaml) file. They have been pre-generated and follow a specific pattern - they start with pre-determined numbers for easier troubleshooting and debugging.
 
-Addresses begin with `YYYXX` number, where `YYY` is a number of an actor and `XX` is [a type of actor](../../shared/core/types/proto/actor.proto#L7). For example, `420043b854e78f2d5f03895bba9ef16972913320` is a validator #420.
-
+Addresses begin with `YYYXX` number, where `YYY` is a number of an actor and `XX` is [a type of actor](../../shared/core/types/proto/actor.proto#L7). For example:
+*  `420043b854e78f2d5f03895bba9ef16972913320` is a validator #420.
+*  `66603bc4082281b7de23001ffd237da62c66a839` is a fisherperson #666.
+*  `0010297b55fc9278e4be4f1bcfe52bf9bd0443f8` is a service node (a servicer) #001.
```

Comment:
Just FYI: We're going to rename all `Service Node` like instances to `Servicer` soon